addons/website_profile/views/website_profile.xml

```diff
@@ -554,7 +554,7 @@
                 <div class="d-inline-block position-relative">
                     <img class="rounded-circle img-fluid"
                         style="width: 128px; height: 128px; object-fit: cover;"
-                        t-att-src="'/profile/avatar/%s?field=image_256%s' % (user['id'], '&amp;res_model=%s&amp;res_id=%s' % (record._name, record.id) if record else '')"/>
+                        t-att-src="'/profile/avatar/%s?field=avatar_256%s' % (user['id'], '&amp;res_model=%s&amp;res_id=%s' % (record._name, record.id) if record else '')"/>
```

Comment:
You need to change the route, `/profile/avatar` does not support this field (and it should actually become the default route value probably).